app/views/articles/show.html.erb

```diff
@@ -147,7 +166,7 @@
             <em> and <b><a href="<%= @third_user.path %>"><%= @third_user.name %></a></b></em>
           <% end %>
           <% if should_show_updated_on?(@article) %>
-            <span><em>Updated on <time itemprop="dateModified" datetime="<%= @article.edited_at&.utc&.iso8601 %>"><%= @article.edited_at&.strftime("%b %d, %Y") %></time></em></span>
```

Comment:
`itemprop="dateModified"` should be deleted, but `datetime="<%= @article.edited_at&.utc&.iso8601 %>"` should stay.